app/controllers/users_controller.rb

```diff
@@ -1,3 +1,34 @@
 class UsersController < ApplicationController
-  def new; end
+  before_action :load_user, only: :show
+
+  def show
+    @user = User.find(params[:id])
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new user_params
+
+    if @user.save
+      flash[:success] = t ".success_mess"
+      redirect_to @user
+    else
+      render :new
```

Comment:
add flash.now[:danger] before this line for warning

Reply:
Done

Reply:
done